scripts/in_container/verify_providers.py

```diff
@@ -360,7 +360,7 @@ def convert_class_name_to_url(base_url: str, class_name) -> str:
     :param class_name: name of the class
     :return: URL to the class
     """
-    return base_url + os.path.sep.join(class_name.split(".")[:-1]) + ".py"
+    return base_url + class_name.rpartition(".")[0].replace(".", os.path.sep) + ".py"
```

Comment:
```suggestion
    return base_url + class_name.rpartition(".")[0].replace(".", "/") + ".py"
```

Not sure if this fits in this PR, but the original code is wrong in using `os.path.sep`. This formats a _URL_ so `/` should always be used instead.